### ISSUE TITLE
system-layout: repeated, or just once?

### ISSUE THREAD

**Issue by webern** [w3c/musicxml#339]
Consider system layout information like this:

```xml
    <measure number="1" width="529.52">
      <print>
        <system-layout>
          <system-margins>
            <left-margin>212.76</left-margin>
            <right-margin>1469.81</right-margin>
            </system-margins>
          <top-system-distance>70.00</top-system-distance>
        </system-layout>
      </print>
    </measure>
```

An application only needs to see this once. It seems like the correct place for this is in the first part only. Or is it recommended that the same layout information should be written in `<measure number="1">` for all parts? Should a recommendation be provided in the documentation? For example, "It is recommended that applications read and write `system-layout` information in the first part only".

**Comment by mdgood** [comment]
Thanks for raising this issue @webern. I think my recommendation would be to read and write any system-specific `<system-layout>` information in the first visible part only. That part could change in each system. I think this makes the document more readable, as the formatting information is confusing if it is written to a part that does not appear on that system. Does that seem reasonable?

**Comment by webern** [comment]
Although it might seem odd for them to be in a hidden part, it seems much easier, whether human or machine, to find them all in the same part, doesn't it? I suppose either way is fine.

**Comment by mdgood** [comment]
@webern Thanks for the feedback. When I was adding support for this feature to the Dolet for Finale plug-in I had to do a lot of debugging to get it working right. I found it easier to deal with these elements in the first visible part, rather than in a part that could easily appear in only a few system in a score, if at all.

How does the wording of #386 look to you? It's phrased a bit differently to fit in more with the overall style. Does it still capture the point clearly?
